spoon-pom/pom.xml

```diff
@@ -20,8 +20,8 @@
     <inceptionYear>2007</inceptionYear>
 
     <properties>
-        <java.src.version>1.8</java.src.version>
-        <java.test.version>1.8</java.test.version>
+        <java.src.version>11</java.src.version>
+        <java.test.version>11</java.test.version>
```

Comment:
Switch to `maven.compiler.release` flag? JDK8 is gone [https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#release](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#release)